src/components/charts/SummaryChartGroup/index.jsx

```diff
@@ -6,6 +6,59 @@ import './SummaryChartGroup.css';
 import helper from '../helper.js';
 
 class SummaryChartGroup extends Component {
+  getChart(item) {
+    switch (item.type) {
+    case 'pie':
```

Comment:
I think the block within switch { } should be indented

Reply:
hmmm I double checked our eslint config file for this repo and portal, and they seems to have a different setting for indentation when comparing to Guppy. Hence, this kind of indentation for case block is valid for ui-component and portal

Reply:
Although I like think indentation rule in Guppy makes more sense, I'll update the eslint config for this repo 👍

Reply:
ok ~

Reply:
Cool! Just fyi I will not change the eslint rule for portal togather with https://github.com/uc-cdis/data-portal/pull/784 because there are just too many switch blocks in portal. So, we should do that as a separate PR 😅